multiautocomplete/src/main/java/com/teamwork/autocomplete/MultiAutoComplete.java

```diff
@@ -131,11 +134,13 @@ public Builder addTypeAdapter(@NonNull AutoCompleteTypeAdapter typeAdapter) {
         /**
          * Build the configured instance of this {@link MultiAutoComplete}.
          */
-        public @NonNull MultiAutoComplete build() {
+        public @NonNull
```

Comment:
Auto-format probably split this into two lines

Reply:
Yup, fixed in next PR along with formatter settings 👍